test/modules/TestResultCoordinator/Program.cs

```diff
@@ -12,15 +15,27 @@ class Program
     {
         static readonly ILogger Logger = ModuleUtil.CreateLogger("TestResultCoordinator");
 
-        static async Task Main()
+        static async Task Main(string[] args)
         {
+            Logger.LogInformation($"Starting load gen with the following settings:\r\n{Settings.Current}");
+
             // TODO: Add Coordinator logic
             (CancellationTokenSource cts, ManualResetEventSlim completed, Option<object> handler) = ShutdownHandler.Init(TimeSpan.FromSeconds(5), Logger);
 
+            await TestOperationResultStorage.Instance.InitAsync(Settings.Current.StoragePath, new SystemEnvironment(), Settings.Current.OptimizeForPerformance, Settings.Current.ResultSources);
+            Console.WriteLine("TestOperationResultStorage fired up");
+            await CreateWebHostBuilder(args).Build().RunAsync(cts.Token);
+            Console.WriteLine("WebHostBuilder fired up");
```

Comment:
Move this line up; otherwise this line will be logged until the web service stopped.